src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/ReferralServiceTest.kt

```diff
@@ -297,22 +297,6 @@ class ReferralServiceTest @Autowired constructor(
     referralService.updateDraftReferral(sampleReferral, DraftReferralDTO(hasAdditionalResponsibilities = true, whenUnavailable = "wednesdays"))
   }
 
-  @Test
-  fun `when usingRarDays is true, maximumRarDays must be set`() {
```

Comment:
Could we add a test please about what happens if the `maximumEnforceableDays` is null when we try to send the referral?

Reply:
sure thing. having said that i'm not sure we explicitly have tests for any other required fields having been set... 

perhaps something to add in another changeset to validate all the required fields?